app/views/_init-containers-summary.html

```diff
@@ -0,0 +1,6 @@
+<span ng-if="$ctrl.podTemplate.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="$ctrl.podTemplate.spec.initContainers.length"
```

Comment:
Nit: can you add something like `class="mar-sm-right"` or `mar-xs-right` so there's a small amount of margin between the text and link?